plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/KuduPageSink.java

```diff
@@ -125,7 +126,10 @@ private KuduPageSink(
             }
 
             try {
-                session.apply(upsert);
+                OperationResponse response = session.apply(upsert);
+                if (response != null && response.hasRowError()) {
```

Comment:
can we extract this verification to a common static method?

`void verifyOperationSucceeded(@Nullable OperationResponse) throw TrinoException`